sktime/forecasting/compose/_reduce.py

```diff
@@ -1352,72 +1352,83 @@ def make_reduction(
         (False: Number of windows = total observations + 1 - window_length
         - forecasting horizon). See pictionary below for more information.
 
-    Please see below a graphical representation of the make_reduction logic using the
```

Comment:
I agree this is the wrong place, but I would actually move it up, not down, to the preamble.

Reply:
Resolved here https://github.com/sktime/sktime/pull/5065/commits/2e31622058ae29448613333495b6d9143d41201a